src/hooks/attendees/useAttendees.ts

```
@@ -0,0 +1,13 @@
+import { useQuery } from "@tanstack/react-query";
+import { getAllAttendees } from "@/data/firestore/attendees";
+import { AttendeeID } from "@/types/sessionTypes";
+import { s } from "framer-motion/dist/types.d-DDSxwf0n";
```

ESLint

[error] 's' is defined but never used.

(@typescript-eslint/no-unused-vars)